db/controller.js

```diff
@@ -8,4 +8,10 @@ exports.getAllTopics = (req, res, next) => {
     .catch((err) => {
         next(err);
       });
-}
\ No newline at end of file
+}
+
+exports.getEndpointDescriptions = (req, res, next)=>{
+    const endpoints = fetchEndpointDescriptions()
+    console.log(fetchEndpointDescriptions())
```

Comment:
don't forget to remove stray console.logs

Reply:
sorry, I thought I got them all!